test/example.t/run.t

```diff
@@ -2,14 +2,14 @@ The driver works on compiled files:
 
   $ ocamlc -I b -I a b/b.mli b/b.ml a/a.mli a/a.ml
 
-  $ odocmkgen -L . -D . > Makefile
+  $ odocmkgen -L a -L b -D a > Makefile
```

Comment:
This test case still works but `a` and `b` must be given explicitly because we no longer search recursively.

Reply:
The `-D` is still not used, this is not new from this PR.